README.md

```diff
@@ -1,4 +1,32 @@
-# Github Templates
+# .github
 
-This repo hosts a few templates to be applied across all repos in this organisation.
-It also holds the organisation's profile, which is public to everyone.
+This repository contains organization-wide GitHub configurations and community health files for the Overture project. These files serve as default community health files for all Overture repositories that don't have their own specific versions.
+
+## Repository Structure
+
+The repository is organized with the following structure:
+
+```
+.
+├── ISSUE_TEMPLATE/
+│   ├── bug_report.md
+│   ├── feature_request.md
+│   └── custom_template.md
+├── workflows/
+├── profile/
+├── Standards/
+├── CODE_OF_CONDUCT.md
+├── CONTRIBUTING.md
+├── CODEOWNERS
+├── FUNDING.yml
+├── PULL_REQUEST_TEMPLATE.md
+├── README.md
+└── ROADMAP.md
+```
+
+- `ISSUE_TEMPLATE/`: Contains templates for different types of issues.
+- `workflows/`: Can hold GitHub Actions workflow files for CI/CD.
+- `profile/`: Contains the files used for the organization landing page readme found on https://github.com/overture-stack.
+- `Standards/`: Includes organizational documentation standards and guidelines.
```

Comment:
I've noticed in other repos these bullet points have initial capitals here regardless of the actual casing of the folder above. this should be consistent everywhere

Reply:
capitalization are reflective of actual casing of the folder/file referenced ` ` is enough to reflect that